app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -14,10 +14,6 @@ export default class IpCameraService extends BaseService {
     return TYPE;
   }
 
-  getImageList() {
-    return this.get('image_list');
-  }
```

Comment:
I totally see your point and agree with you, I'm just waiting for more stable API state. We just did several things and didn't have time to bulletproof API in full, IMO there is high probability that some big pieces can change. But once we know that API reaches some milestone - then no doubt we'll need to expose methods that we don't use, but others can.